packages/backend/src/routers/apiV1Router.ts

```diff
@@ -158,14 +157,12 @@ apiV1Router.get(
 
 apiV1Router.get(lightdashConfig.auth.google.callbackPath, (req, res, next) => {
     passport.authenticate('google', {
-        failureRedirect: '/api/v1/oauth/failure',
-        successRedirect: getSuccessURLWithReturnTo(req),
+        failureRedirect: getOidcRedirectURL(false)(req),
+        successRedirect: getOidcRedirectURL(true)(req),
         failureFlash: true,
         includeGrantedScopes: true,
     })(req, res, next);
 });
-apiV1Router.get('/oauth/failure', redirectOIDC);
-apiV1Router.get('/oauth/success', redirectOIDC);
```

Comment:
Before we had 3 redirects:
```
# oauth redirects to
http://localhost:3000/api/v1/oauth/redirect/google?error=access_denied&state=rrpyagokHaoxSjOyntMYIvS7

# passport oauth redirects to
http://localhost:3000/api/v1/oauth/failure

# this route redirects to 
http://localhost:3000/
```

I moved the redirect logic to the passport oauth to avoid the extra redirect.
The redirect logic is now on `getOidcRedirectURL` method